includes/functions/llms-functions-content.php

```diff
@@ -112,6 +112,24 @@ function llms_get_post_sales_page_content( $post, $default = '' ) {
 
 }
 
+/**
+ * Retrieve the decimal value for average column in Reporting.
+ *
+ * By default the database return floating-point values and  function
+ * percision to round off the decimal value.
```

Comment:
this sentence sound a bit weird to me, also two spaces between `and` and `function` are too many :D

Reply:
my bad, I forgot to update this sentence. Now fixed.